tests/test_ex2.py

```diff
@@ -0,0 +1,6 @@
+from src.ex2 import ex2
+
+def test_ex2() -> None:
+	assert ex2(5) == "Thursday"
+	assert ex2(2) == "Monday"
+	assert ex2(1) == "Sunday"
```

Comment:
you should cover the side effect, i recommend you to using [pytest-cov](https://pypi.org/project/pytest-cov/) and to see if the coverage of your test has a reasonable percentage close to 100% of your code